packages/react-pages/src/node/index.ts

```diff
@@ -214,24 +216,25 @@ export { extractStaticData, File } from './utils/virtual-module'
 export { PageStrategy }
 export { DefaultPageStrategy, defaultFileHandler }
 
-async function getRemarkPlugins(root: string) {
+async function getRemarkPlugins() {
   const result: any[] = [
     DemoMdxPlugin,
     TsInfoMdxPlugin,
     ImageMdxPlugin,
     FileTextMdxPlugin,
   ]
-  const pkgJsonPath = path.join(root, 'package.json')
-  // TODO: user may put the whole vite-pages project
-  // under a sub folder (which is the root here),
-  // so the package.json will be located at the upper folder.
-  // checkout playground/custom-find-pages2.
-  const hasPkgJson = fs.pathExistsSync(pkgJsonPath)
+
+  const pkgJsonPath = pkgUp.sync()
```

Comment:
You should pass the `root` to it, otherwise it will start from `process.cwd()`: https://www.npmjs.com/package/pkg-up

We want it to start from vite's root.